src/loaddb/load_session.cpp

```diff
@@ -242,13 +246,43 @@ namespace cubload
 
 	// notify session that batch is done
 	m_session.notify_batch_done (m_batch.get_id ());
+
+	thread_ref.m_loaddb_driver->clear ();
       }
 
     private:
       const batch &m_batch;
       session &m_session;
+      css_conn_entry &m_conn_entry;
   };
 
+  static loaddb_worker_context_manager *loaddb_wp_context_manager;
+  static cubthread::entry_workpool *loaddb_worker_pool;
+  static std::mutex loaddb_wp_mutex;
+  static uint64_t loaddb_sessions_started;
```

Comment:
Unsigned int should be enough, no reason for 64-byte variable. It is better to initialize count as 0.

```suggestion
  static unsigned int loaddb_sessions_started = 0;
```